pkg/configuration/configuration.go

```diff
@@ -112,37 +123,54 @@ const (
 	// varTwilioAccountSID is the constant used to read the configuration parameter for the
 	// Twilio account identifier, used for sending SMS verification codes.  Twilio is a service that
 	// provides an API for sending SMS messages anywhere in the world.  http://twilio.com
-	varTwilioAccountSID = "twilio.account_sid"
+	varTwilioAccountSID = "twilio.account.sid"
 
 	// varTwilioAuthToken is the constant used to read the configuration parameter for the
 	// Twilio authentication token, used for sending SMS verification codes
-	varTwilioAuthToken = "twilio.auth_token"
+	varTwilioAuthToken = "twilio.auth.token"
 )
 
 // Registry encapsulates the Viper configuration registry which stores the
 // configuration data in-memory.
 type Registry struct {
```

Comment:
Btw, we don't use the `Registry` name in any of our operators any more. I believe that it would be worth renaming here it too.

Reply:
@MatousJobanek sounds good, i guess in host-operator we use "Config" - is this fine to use?

Reply:
Yup :+1:

Reply:
done as part of: 1863479426aebd4c11524c081a17b3773e64e32c